reactome_analysis_api/reactome_analysis_api/__main__.py

```diff
@@ -27,7 +27,7 @@
 app.add_api('swagger.yaml', arguments={'title': 'REACTOME Analysis Service'})
 
 # create the object for the public data searcher
-app.app.public_searcher = PublicDatasetSearcher(path=os.getenv("SEARCH_INDEX_PATH", "../"))
+app.app.public_searcher = PublicDatasetSearcher(path=os.getenv("SEARCH_INDEX_PATH", "../"), path_whitelist=os.getenv("SEARCH_INDEX_WHITELIST", "../"))
```

Comment:
Shouldn't this path point to the actual text file? If yes, then the default value should also be a specific filename.

Reply:
changed to specific file "whitelist.txt"